.travis.yml

```diff
@@ -4,12 +4,12 @@ sudo: false
 
 python:
   - 2.7
```

Comment:
We still want to tests 2.7 with NUMPY_VERSION=1.11.3 PANDAS_VERSION=0.18.1. (Taken from [zipline](https://github.com/quantopian/zipline/blob/013cd1b0ef916024a7233301a2c127111e889434/.travis.yml#L47))

Reply:
The travis CI build works, but the appveyor build is bugged. I can change the travis build, but the appveyor build won't work until the underlying bug gets fixed.

Also, I would suggest updating zipline to require newer versions of both pandas and numpy.

Reply:
> the appveyor build is bugged

Oh, is it a particular job in the matrix that's broken?

> Also, I would suggest updating zipline to require newer versions of both pandas and numpy.

Agreed - PRs welcome 😄 .

Reply:
In appveyor, the pandas version is not constrained correctly from the specification, so it tries using pandas 0.24.1 with numpy 1.11.3, which are incompatible.

Reply:
Ah, looks like the `conda create -n testenv` step is missing `pandas=%PANDAS_VERSION%`

Reply:
Thanks, and fixed (: